recipe/meta.yaml

```diff
@@ -96,8 +94,6 @@ test:
     - conda-render --help
     - conda-skeleton --help
     - conda-debug --help
-    # bdist_conda
-    - python test_bdist_conda_setup.py bdist_conda --help
```

Comment:
The custom `bdist_conda` `distutils`/`setuptools` function was deprecated in 24.3.0 (see https://github.com/conda/conda-build/issues/4985, https://github.com/conda/conda-build/pull/5196) and removed in 24.5.0 (see https://github.com/conda/conda-build/pull/5299)

Reply:
Thank you for making this change, @kenodegard ! Could you also please remove the `recipe/test_bdist_conda_setup.py` test file?

Reply:
whoops!